scapy/layers/ipsec.py

```diff
@@ -47,8 +47,17 @@
 from scapy.config import conf, crypto_validator
 from scapy.data import IP_PROTOS
 from scapy.error import log_loading
-from scapy.fields import (ByteEnumField, ByteField, StrField, StrLenField,
-                          XIntField, IntField, ShortField, PacketField)
+from scapy.fields import (
+    ByteEnumField,
+    ByteField,
+    IntField,
+    PacketField,
+    ShortField,
+    StrField,
+    XIntField,
+    XStrField,
+    XStrLenField,
+)
```

Comment:
It is a question of point of view ^^
I find this clearer, easier to sort and make diff shorter afterwards.
I can change though, it is quite easy!